cpp/csp/python/PyStructFastList_impl.h

```diff
@@ -511,6 +511,27 @@ static PyObject * PyStructFastList_Reversed( PyStructFastList<StorageT> * self,
     CSP_RETURN_NULL;
 }
 
+template<typename StorageT>
+static PyObject * PyStructFastList_reduce( PyStructFastList<StorageT> * self, PyObject * Py_UNUSED( ignored) )
+{
+    CSP_BEGIN_METHOD;
+    
+    typename VectorWrapper<StorageT>::IndexType sz = self -> vector.size();
+    PyObject * args = PyTuple_New( sz );
+    PyObjectPtr list = PyObjectPtr::own( toPython( self -> vector.getVector(), self -> arrayType ) );
+    for( typename VectorWrapper<StorageT>::IndexType index = 0; index < sz; ++index )
+    {
+        PyObject * value = PyList_GET_ITEM( list.ptr(), index );
+        Py_INCREF( value );
+        PyTuple_SET_ITEM( args, index, value );
+    }
+    PyObject * result = Py_BuildValue( "O(O)", &PyList_Type, args );
+    Py_INCREF( result );
```

Comment:
dont see why you would need an incref here

Reply:
Removed the INCREF